packages/app-builder/src/components/ui/render-node/styles.ts

```diff
@@ -0,0 +1,47 @@
+import { css } from '@linaria/core'
+
+export const indicator = css`
+  height: 30px;
+  margin-top: -29px;
+  font-size: 12px;
+  line-height: 12px;
+  color: white;
+  position: fixed;
+  background: var(--intent-primary);
+
+  svg {
+    fill: #fff;
+    width: 15px;
+    height: 15px;
+  }
+`
+
+export const componentSelected = css`
+  position: relative;
+  &:after {
+    content: ' ';
```

Comment:
```suggestion
    content: '';
```
don't need a space :)

Reply:
TIL!